modules/include/cgi/ACGI.hpp

```diff
@@ -0,0 +1,329 @@
+#pragma once
+
+#include <any>
+#include <array>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <regex>
+#include <sstream>
+#include <unordered_map>
+
+#include "ziapi/Config.hpp"
+#include "ziapi/Logger.hpp"
+#include "ziapi/Module.hpp"
+
+using namespace ziapi;
+
+#if defined(WIN32) || defined(_WIN64) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__)
+#define CGI_WIN
+#include <strsafe.h>
+#include <windows.h>
+#define POPEN _popen
+#define PCLOSE _pclose
+#define FGETS fgets
+#else
+#define POPEN popen
+#define PCLOSE pclose
+#define FGETS fgets
+#endif
+
+class ACGI : public IHandlerModule {
+protected:
+    static const inline std::string kHeaderBodySeparator = "\r\n\r\n";
```

Comment:
maybe use `std::string_view` instead
```suggestion
    static constexpr std::string_view kHeaderBodySeparator = "\r\n\r\n";
```